libraries/Microsoft.Bot.Builder.LanguageGeneration/Evaluator.cs

```diff
@@ -21,11 +21,14 @@ namespace Microsoft.Bot.Builder.LanguageGeneration
     /// </summary>
     public class Evaluator : LGTemplateParserBaseVisitor<object>
     {
+        /// <summary>
+        /// A symbol of a string constant "lgType".
+        /// </summary>
         public const string LGType = "lgType";
```

Comment:
> public const string LGType = "lgType"; [](start = 7, length = 39)

It is weird this symbol is defined both in Evaluator and Expander.  Are they the same?  If so, only one place should have them.  Is this used publically?

Reply:
After talking to other team members, the symbol in Expander has been change to private. And the one in Evaluator has been changed to internal.